src/main/java/com/ericsson/ei/subscriptionhandler/SubscriptionValidator.java

```diff
@@ -0,0 +1,137 @@
+/*
+   Copyright 2017 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.subscriptionhandler;
+
+//import java.util.List;
+import java.util.regex.Pattern;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.ericsson.ei.controller.model.Subscription;
+import com.ericsson.ei.exception.SubscriptionValidationException;
+//import com.ericsson.ei.controller.model.Requirement;
+
+public class SubscriptionValidator {
+	
+	private static final Logger LOG = LoggerFactory.getLogger(SubscriptionValidator.class);
+	
+	/*
+	 * Validation of parameters values in subscriptions objects.
+	 * Throws SubscriptionValidationException if validation of a parameter fails due to wrong
+	 * format of parameter.
+	 * 
+	 * @param Subscription 
+	 */
+	public void validateSubscription(Subscription subscription) throws SubscriptionValidationException {
+		
+		LOG.info("Validation of subscription " + subscription.getSubscriptionName() + " Started.");
+		
+		this.validateSubscriptionName(subscription.getSubscriptionName());
+		this.validateNotificationMessage(subscription.getNotificationMessage());
+		this.validateNotificationMeta(subscription.getNotificationMeta());
+		this.validateNotificationType(subscription.getNotificationType());
+		this.validateRepeat(subscription.getRepeat());
+//		List<Requirement> reqList = subscription.getRequirements();
+//		for (int i=0; i < reqList.size(); i++) {
+//			this.validateJmespath(reqList.get(i).getConditions().get(0).getJmespath());
+//		}
```

Comment:
I have comment out Jmepath part, since it do not validate anything yet. Need to find a good way to validate JmePath syntax. This can be done in a new story. Shall I remove the commented out Jmepath code or keep it comment out until JmePath syntax validator is implemented?